File: c/functions/problem_1.c
#include <stdio.h>

float average(float, float, float);

float average(float x, float y, float z)
{
    return (x + y + z) / 3;
}

int main()
{
    float a, b, c;
    printf("Enter the 1st number : \n");
    scanf("%f", &a);
    printf("Enter the 2nd number : \n");
    scanf("%f", &b);
    printf("Enter the 3rd number : \n");
    scanf("%f", &c);

    printf("The average of these three numbers is %.2f \n", average(a, b, c));

    return 0;
}

File: python/conditionals/problem1.py
a=int(input("Enter a :"))
b=int(input("Enter b :"))
c=int(input("Enter c :"))
d=int(input("Enter d :"))

if(a>b and a>c and a>d):
    print("a is the greatest number")
elif(b>c and b>d):
    print("b is the greatest number")
elif(c>d):
    print("c is the greatest number")
else :
    print("d is the greatest number")            